src/components/funFactPage/funFactPage.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
1. The convention is this project is to name our component files with a capital letter in the beginning, like `FunFactPage.tsx` and `FunFactPage.scss`. Sadly, Windows file systems are case insensitive, so _this can't be fixed_ without having to delete a whole bunch of stuff and recreate it 😬 you can leave this one, but please try to remember in future! (The folder name, `funFactPage`, is cased correctly 😊 although you may be able to fix the casing by renaming this file with a capital (`funFactPage.tsx -> FunFactPage.tsx`), then renaming the folder to `funFacts`, and committing them both together, so try that! If it doesn't work don't worry)
2. Don't create an empty `.scss` file just as a placeholder, we may not use it! Please delete that file :blush:

Reply:
Done - I think the foldername update funFacts worked so I hope that comes through